include/sys/trace_dmu.h

```diff
@@ -62,7 +62,7 @@ DECLARE_EVENT_CLASS(zfs_delay_mintime_class,
 	    __entry->tx_lastsnap_txg		= tx->tx_lastsnap_txg;
 	    __entry->tx_lasttried_txg		= tx->tx_lasttried_txg;
 	    __entry->tx_anyobj			= tx->tx_anyobj;
-	    __entry->tx_waited			= tx->tx_waited;
+	    __entry->tx_waited			= tx->tx_dirty_delayed;
```

Comment:
The `printk` below looks like it's attempting to print out the fields of the `dmu_tx_t` structure. I think you should rename `tx_waited` to be `tx_dirty_delayed` in the in the `__entry` structure to be consistent.

Reply:
Yes, please update all the instances of `tx_waited` in this macro to `tx_dirty_delayed`.